docs/src/developers_guide/contributing_pytest_conversions.rst

```diff
@@ -41,7 +41,7 @@ Conversion Checklist
 
 #. Check for references to ``@tests``. These should be changed to ``@_shared_utils``.
 #. Check for references to ``with mock.patch("...")``. These should be replaced with
-   ``mocker.patch("...")``. Note, ``mocker.patch("...")`` is NOT a context manager.
+   ``mocker.patch("...")``, which can be passed into functions as a fixture.
```

Comment:
I don't understand this phrasing. Are you saying tests should look like this:

```python
def test_something(mocker.patch):
    mocker.patch(...)
```

Or like this:

```python
def test_something(mocker):
    mocker.patch(...)
```

Reply:
mocker is the fixture here, so the latter. 

```suggestion
   ``mocker.patch("...")``. ``mocker`` is a fixture, and be passed into functions.
```